From 85c98487cd805d5194f4a4327bb2cf35dd2ffc94 Mon Sep 17 00:00:00 2001
Message-Id: <patch-10.12-85c98487cd8-20220224T092611Z-avarab@gmail.com>
In-Reply-To: <cover-00.12-00000000000-20220224T092611Z-avarab@gmail.com>
References: <cover-00.12-00000000000-20220224T092611Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sat, 19 Feb 2022 22:26:07 +0100
Subject: [PATCH 10/12] refspec.c: use designated initializers for "struct
 refspec_item"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the "struct refspec_item" at the top of refspec.c to use
designated initializers. Let's keep the "= 0" assignments for
self-documentation purposes, even though they're now redundant.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 refspec.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/refspec.c b/refspec.c
index e3d852c0bfe..63e3112104a 100644
--- a/refspec.c
+++ b/refspec.c
@@ -4,13 +4,13 @@
 #include "refspec.h"
 
 static struct refspec_item s_tag_refspec = {
-	0,
-	1,
-	0,
-	0,
-	0,
-	"refs/tags/*",
-	"refs/tags/*"
+	.force = 0,
+	.pattern = 1,
+	.matching = 0,
+	.exact_sha1 = 0,
+	.negative = 0,
+	.src = "refs/tags/*",
+	.dst = "refs/tags/*",
 };
 
 /* See TAG_REFSPEC for the string version */
-- 
2.35.1.1157.g524e2d5a0db

